Vendor.Application/Requests/Services/AddServiceCommand.cs

```diff
@@ -0,0 +1,43 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Vendor.Domain.Entities;
+
+
+namespace Vendor.Application.Requests.Services
+{
+    public class AddServiceCommand:IRequest<Service>
+    {
+        public string Name { get; set; }
+    }
+    public class AddServiceCommandHandler : IRequestHandler<AddServiceCommand, Service> 
+    {
+        private readonly VendorDbContext _vendorDbContext;
+        public AddServiceCommandHandler(VendorDbContext vendorDbContext)
+        {
+            _vendorDbContext = vendorDbContext;
+        }
+        public async Task<Service> Handle(AddServiceCommand request,CancellationToken cancellationToken)
+        {
+            if (string.IsNullOrWhiteSpace(request.Name))
```

Comment:
LLD is missing. Add LLD in all handler methods

Reply:
done